examples/lint.sh

```diff
@@ -14,9 +14,10 @@ check_files() {
     local depth=1
     local -a extensions=("-name" "*.json" -o "-name" "*.yml" -o "-name" "*.yaml" -o "-name" "*.graphql" -o "-name" "*.gql")
     local command="./target/debug/tailcall check"
+    local -a ignore=("!" "-name" "grpc-reflection.graphql")
```

Comment:
Why don't we want to lint this file?

Reply:
This avoids running check on the file. We need to run reflection server in order to run `tc check` on reflection config